src/main/java/ru/fmt/university/rest/controller/GroupController.java

```diff
@@ -19,9 +24,13 @@
     private GroupService groupService;
 
     @PostMapping(value = "/groups")
-    public ResponseEntity<?> create(@RequestBody Group group) {
-        groupService.create(group);
-        return new ResponseEntity<>(HttpStatus.CREATED);
+    public ResponseEntity<Group> create(@Validated(Create.class) @RequestBody Group group, BindingResult bindingResult) {
+        Group created = groupService.create(group);
+
+        if (bindingResult.hasErrors()) {
+            return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
```

Comment:
you should also add a message, not a simple 400 code. In other case I as user would not know where a make mistake
https://stackoverflow.com/questions/32441919/how-return-error-message-in-spring-mvc-controller